Makefile

```diff
@@ -354,8 +356,11 @@ check: test
 install: $(wildcard *.go)
 	$(GO) install -v -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)'
 
+.PHONY: go
+go: go-check $(EXECUTABLE)
+
 .PHONY: build
-build: $(EXECUTABLE)
+build: node-check go js css
```

Comment:
I think `js` and `css` need to be built first for `bindata.go` to be created properly

Reply:
Indeed, I think you're right.